tests/integration/p2p.go

```diff
@@ -479,7 +337,61 @@ func getAllP2PCollections(
 	assert.Equal(s.t, expectedCollections, cols)
 }
 
-// waitForSync waits for all given wait channels to receive an item signaling completion.
+// waitForUpdateEvents waits for all selected nodes to publish an update event.
```

Comment:
suggestion: Building on the other related documentation request, it might be worth highlighting that you mean publish to the local event queue, not to the pub/sub net or some other P2P related network thing.